ska_helpers/tests/test_git_helpers.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import tempfile
+from ska_helpers import git_helpers
+import git
+import os
+from pathlib import Path
+
+
+CHANDRA_MODELS = Path(os.environ["SKA"]) / "data" / "chandra_models"
+
+
+def ska_ownership_ok():
+    try:
+        return CHANDRA_MODELS.owner() == os.getlogin()
+    except Exception:
+        return False
+
+
+@pytest.mark.skipif(not CHANDRA_MODELS.exists(), reason="Chandra models dir is not there")
+@pytest.mark.skipif(ska_ownership_ok(), reason="Chandra models dir ownership is OK")
+def test_make_git_repo_safe():
+    with tempfile.TemporaryDirectory() as tempdir, pytest.warns(
+        UserWarning, match="Updating git config"
+    ):
+        os.environ["HOME"] = tempdir
```

Comment:
This should use the `monkeypatch.setenv()` fixture to temporarily change `$HOME`. Otherwise I think this is a persistent side effect of this test.

Reply:
ah, I just fixed it the longwinded way, I will use monkeypatch instead

Reply:
Fixed